### DISCUSSION TITLE
Add more rapid tools/seeds/resources

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1301] [Ideas]
This is my idea

![Schermata 2022-08-02 alle 12 05 46](https://user-images.githubusercontent.com/58088988/182349509-3c07a78e-78c9-4d23-aecc-8bc25a1c6ed7.png)

- Quatrefoil
- Farm ID : 122927

**Comment by adamhannigan** [comment]
@Q4FOIL One of our limitations is that the game is built for mobile-first.

How do you think this would look on a mobile screen? (especially as we add more HUD buttons around the screen)

**Comment by Q4FOIL** [comment]
@adamhannigan so actually your goal is to have fewer objects present ..

What do you think of the mobile menu system?

I'll give you an example:

Using the inventory we have a global set.
You could create these two quick accesses for tools and seeds simply by making them drop down.

Let me explain even better

https://user-images.githubusercontent.com/58088988/182525497-d91bb2c0-df3d-40e7-8f48-44b71ab9ee60.mov

_Obviously it is assumed that only the objects that you own will appear inside and not those that you could possess_
